### PR TITLE
Only show representations with >1 submissions

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

File: app/commands/exercise/representation/search.rb [hunk 1]
```diff
@@ -10,7 +10,9 @@ def self.requests_per_page = 20
   end
 
   def call
-    @representations = Exercise::Representation.includes(:exercise, :track)
+    @representations = Exercise::Representation.
+      includes(:exercise, :track).
+      where('num_submissions > 1')
```
Comment (ErikSchierboom) [inline app/commands/exercise/representation/search.rb:15]: How does this work with indexing?

Comment (iHiD) [reply app/commands/exercise/representation/search.rb:15]: On filtering its faster by about 80ms. On count, it's slower by about 2ms.

🤷

Comment (ErikSchierboom) [reply app/commands/exercise/representation/search.rb:15]: Okay.

Comment (ErikSchierboom) [reply app/commands/exercise/representation/search.rb:15]: Do we need to update our indexes?

**Review by ErikSchierboom — APPROVED** [review]
LGTM

File: app/commands/exercise/representation/search.rb (hunk 1 above)
Comment (iHiD) [reply app/commands/exercise/representation/search.rb:15]: No. It's just fast 🤷 

I mean, adding extra indexes might help but it's not slower without them from what I can tell.

**Comment by iHiD** [comment]
@ErikSchierboom Any chance you could get this over the line. I've tried 4 times to get it green and am failing. It'll just be changing a total_count down 1, or something probably. Thanks!

**Comment by ErikSchierboom** [comment]
Should be fixed with: https://github.com/exercism/website/pull/2985/commits/b14c079044d158dab6cc7b60389316f0c7d80ff6
I also added a test for the num_submissions filtering: https://github.com/exercism/website/pull/2985/commits/462d714c159804c89a5db3de6393c89634996570

**Comment by iHiD** [comment]
Thank you!! 💙

**Pull request merged**